### PR TITLE
Configure Hurl BSS Integration Tests

### PR DESCRIPTION
**Pull request by synackd**
The BSS integration tests were originally in the [BSS](https://github.com/OpenCHAMI/bss) repository and were written to be run serially. They were then moved here in their state at the time when they were copied. This PR adjusts them to be able to be run in parallel using Docker Compose.

### CODE REVIEW

**Comment by synackd** [comment]
The merge of #13 was useful for getting this fleshed out, and, after a rebase with some additional commits, it is finally ready to be merged.

File: lanl/docker-compose/tests/all/01b-smd-add-redfish-endpoints-nids.hurl [hunk 1]
```diff
@@ -1408,64 +1408,67 @@ POST http://smd:27779/hsm/v2/Inventory/RedfishEndpoints
 HTTP 201
 Content-Type: application/json
 [Asserts]
-jsonpath "$[0].URI" == "/hsm/v2/Inventory/RedfishEndpoints/x0c0s3b0"
+jsonpath "$[0].URI" == "/hsm/v2/Inventory/RedfishEndpoints/x0c1s3b0"
 
+#
 # Get RedfishEndpoints we just added.
-GET http://smd:27779/hsm/v2/Inventory/RedfishEndpoints
```
Comment (davidallendj) [inline lanl/docker-compose/tests/all/01b-smd-add-redfish-endpoints-nids.hurl:1414]: Getting rid of the base cases?

Comment (synackd) [reply lanl/docker-compose/tests/all/01b-smd-add-redfish-endpoints-nids.hurl:1414]: Ah, I presume you mean checking `v2/Inventory/RedfishEndpoints` as well as `v2/Inventory/RedfishEndpoints/{xname}`. That is a good point, and I will add that as a test case.

Comment (synackd) [reply lanl/docker-compose/tests/all/01b-smd-add-redfish-endpoints-nids.hurl:1414]: Added in 861e634.

Comment (synackd) [reply lanl/docker-compose/tests/all/01b-smd-add-redfish-endpoints-nids.hurl:1414]: For RedfishEndpoints, Components, and ComponentEndpoints.

**Review by davidallendj — APPROVED** [review]

**Comment by synackd** [comment]
LGTM

**Pull request merged**